crates/e2e/src/zombienet.rs

```diff
@@ -0,0 +1,42 @@
+use crate::get_contract_node_bin;
+use zombienet_sdk::{
+    NetworkConfig,
+    NetworkConfigBuilder,
+    RegistrationStrategy,
+};
+
+/// Build a network configuration with the default environment variables.
+pub fn build_network_with_env_or_default() -> NetworkConfig {
+    let missing_polkadot_binaries = [
+        "polkadot",
+        "polkadot-execute-worker",
+        "polkadot-prepare-worker",
+    ]
+    .into_iter()
+    .filter(|binary| which::which(binary).is_err())
+    .collect::<Vec<_>>();
+
+    if !missing_polkadot_binaries.is_empty() {
+        panic!("The following binaries were not found in the PATH: {missing_polkadot_binaries:?}");
+    }
```

Comment:
@pepoviola do we have these checks when we configure the Network with "polkadot" command? This could be removed here if zombienet_sdk panic with similar error messages

Reply:
If the `polkadot` is not in the path the spawn will fail but with a different error, you are asserting on the error?

Reply:
I just want the error to be easy to find for users trying to run their tests.
I'll try to hit the error to see how it looks when these binaries are missing.

Reply:
IIRC if `polkadot` is missing the spawn will fail quickly but if one of the workers is missing the failure will be a little bit more hidden in the `node`'s logs.